net/portmapper/portmapper.go

```diff
@@ -154,6 +157,9 @@ func (m *pmpMapping) Release(ctx context.Context) {
 // goroutine whenever the port mapping status has changed. If nil,
 // it doesn't make a callback.
 func NewClient(logf logger.Logf, onChange func()) *Client {
+	// Initialize the debug knobs if they're not set yet.
```

Comment:
Any chance we make the debug knobs be `NewClient` parameters instead (and store them on the `Client` instance instead of in globals). It seems like in all the places they're accessed we have either direct or indirect access to the `Client` to read them from.

Reply:
Updated 👍